socorro/processor/general_transform_rules.py

```diff
@@ -22,22 +22,25 @@ def version(self):
         return '1.0'
 
     def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-        processed_crash.cpu_info = ''
-        processed_crash.cpu_name = ''
+        cpu_name = ''
+        cpu_info = ''
 
-        system_info = processed_crash.json_dump.get('system_info')
+        system_info = processed_crash.get('json_dump', {}).get('system_info')
         if system_info:
-            processed_crash.cpu_name = system_info.get('cpu_arch', '')
-            try:
-                processed_crash.cpu_info = (
+            cpu_name = system_info.get('cpu_arch', '')
+
+            if 'cpu_info' in system_info and 'cpu_count' in system_info:
+                cpu_info = (
                     '%s | %s' % (
                         system_info['cpu_info'],
                         system_info['cpu_count']
                     )
                 )
```

Comment:
I don't generally use `str.format`. It's super helpful in some specific cases, but `%s` is generally safer.